src/misc/DeviceConfig.ts

```diff
@@ -43,6 +49,8 @@ interface ConfigObject {
 	syncContactsWithPhonePreference: Record<Id, boolean>
 	/** Whether mobile calendar navigation is in the "per week" or "per month" mode */
 	isCalendarDaySelectorExpanded: boolean
+	/** Stores user's desired behavior to the view when an email is removed from the list */
+	listAutoSelectBehavior: ListAutoSelectBehavior
```

Comment:
I think the field should be named "mailAutoSelectBehavior" since it's about mail specifically. But only here, not in List itself or the type because that one is applicable generally. This field is only about mails tho.